packages/ag-charts-enterprise/src/features/annotations/annotationOptionsToolbar.css

```diff
@@ -12,6 +12,28 @@
     opacity: 0.5;
 }
 
+.ag-charts-annotations__color-picker-button[aria-disabled='true']::after {
+    filter: grayscale(1);
+    opacity: 0.5;
```

Comment:
Should it be a greyscaled gradient, or a solid colour? It might be easier to just set `--emblem` to a grey colour